pkgs/misc/my-env/default.nix

```diff
@@ -54,10 +54,19 @@
   # The result using that command should be:
   #  env-nix loaded
   and show you a shell with a prefixed prompt.
-*/
+
+  # The prompt of the new shell can be defined by setting ps1 in the
+  # arguments to myEnvFun.  The default value is
+  # ps1 = "\\n\\$NIX_MYENV_NAME:[\\u@\\h:\\w] $ "
+  # Note that 2 backslashes are needed to pass an escape which is recognized in
```

Comment:
Minor comment: Could you put something like "The PS1 variable will be inserted *unescaped* into the initialization script wrapped with double quotes: so don't put untrusted input here; variable substitution will work; and backslashes and double quotes must be escaped.

(Is there an easy way to escape it? That would remove the $NIX_MYNV_NAME feature, but whoever is calling this function could probably just put in ${name} and have nix do the substituting.)